linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_ip_local_port_range/tests/comment.fail.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if grep -q "^net.ipv4.ip_local_port_range" /etc/sysctl.conf; then
+	sed -i "s/^net.ipv4.ip_local_port_range.*/# net.ipv4.ip_local_port_range = 32768 65535/" /etc/sysctl.conf
+else
+	echo "# net.ipv4.ip_local_port_range = 32768 65535" >> /etc/sysctl.conf
+fi
+# setting correct runtime value to check it properly
```

Comment:
This comment is a bit misleading.
The test comments the file config, but sets correct runtime values.
Suggestion: 
` # setting correct runtime value to check if the filesystem configuration is evaluated properly`